### PR TITLE
WSTEAM1-139 Use Image Component in Bylines

### PR DESCRIPTION
**Pull request by mfonofm**
Resolves #WSTEAM1-139

**Overall change:**
Use `Image` component in bylines and add validation to check if image is in `.png` format

**Code changes:**

- Use `Image` component
- Add validation to ensure we only render `.png` images
- Add fixture data
- Add and update unit tests
- Use `buildIChefURL` function

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by greenc05** [comment]
When CSS is off the image is very large, can this be smaller (in both dimensions and file size)? Actually think I am looking at an old storybook link and this might be resolved now with https://github.com/bbc/simorgh/pull/10319/files#diff-c6e2510a27f953a0f94d7145f20ca188665b6e9f3b80cea08c9a2d559690a613R126

File: src/app/pages/ArticlePage/Byline/index.styles.tsx [hunk 1]
```diff
@@ -103,14 +103,16 @@ export default {
...
     css({
-      display: 'block',
+      display: 'inline-flex',
+      alignItems: 'flex-end',
+      justifyContent: 'center',
       width: `${pixelsToRem(80)}rem`,
-      height: `${pixelsToRem(80)}rem`,
+      height: `${pixelsToRem(60)}rem`,
       border: `solid ${pixelsToRem(1)}rem #979797`,
       backgroundColor: '#d8d8d8',
```
Comment (greenc05) [inline src/app/pages/ArticlePage/Byline/index.styles.tsx:114]: Wondering why we have a background colour?

**Review by Louis-Matsika — APPROVED** [review]

File: src/app/pages/ArticlePage/Byline/index.tsx [hunk 2]
```diff
@@ -116,11 +119,16 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
...
-                ? [BylineCss.imageRtl, BylineCss.Image]
-                : [BylineCss.imageLtr, BylineCss.Image]
+                ? [BylineCss.imageRtl, BylineCss.ImageWrapper]
+                : [BylineCss.imageLtr, BylineCss.ImageWrapper]
             }
           >
-            <img css={BylineCss.imageSource} src={image} alt="" />
+            <Image
+              css={BylineCss.imageSrc}
+              src={image}
```
Comment (amoore108) [inline src/app/pages/ArticlePage/Byline/index.tsx:141]: I think the `src` may need some URL construct applied to it to point to the correct iChef location.

Looking at the data, the `locator` field will only have part of the URL, e.g: `"locator": "f2f8/test/49056060-fea9-11ec-8adf-7bf4017ee9f9.png",` so I think you may need to use the `buildIChefUrl` function found in [#lib/utilities/ichefURL](https://github.com/bbc/simorgh/blob/7e36689744a7d29d286293a887669ec49d8f7db4/src/app/lib/utilities/ichefURL/index.js#L20).

You can pass in a `resolution` to that function as well, so we can prevent really large pngs for this case.

**Review by amoore108 — APPROVED** [review]
Looks good!

**Pull request merged**